Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM openshift/origin-base
+
+ENV APP_HOME=/opt/app/src \
+    GOPATH=/go \
+    PATH="${PATH}:${GOPATH}/bin:/usr/local/bin" \
+    INSTALL_PKGS="golang make git"
+
+RUN yum install -y ${INSTALL_PKGS} --setopt=tsflags=nodocs
+
+WORKDIR /tmp/cluster-logging-operator
+COPY vendor/ /tmp/cluster-logging-operator/vendor
+COPY scripts/* /usr/local/bin
```

Comment:
this is called with a relative path to the operator, I don't think we should move it to the bin path nor make it executable (its called with `bash cert_generation.sh`

Reply:
The operator binary is moved to `/usr/local/bin` so it will be relative to the binary.  I can remove executable bit if you think that's better; I don't have a strong opinion

Reply:
ack, so can we copy it to `/usr/local/bin/scripts/` then? so this way I don't need to update the operator code to use a different path from what it does currently which is `scripts/cert_generation.sh`.

Reply:
Also, I'm alright with leaving the permissions change off. This way someone isn't tempted to just run it?

Reply:
Updated